### PR TITLE
Improve matching, remove uppercase PR #3325

### PR DESCRIPTION
**Pull request by kirkins**
## People to notify

---

Instant Answer Page: https://duck.co/ia/view/{ID}

### CODE REVIEW

File: lib/DDG/Spice/Edrpou.pm [hunk 1]
```diff
@@ -15,12 +15,12 @@ spice wrap_jsonp_callback => 1;
 spice to => 'https://edr.data-gov-ua.org/api/companies?where={%22edrpou%22:%22$1%22}';
 
 # Triggers - https://duck.co/duckduckhack/spice_triggers
-triggers startend => "EDRPOU","ЄДРПОУ","ЄДР","edrpou", "єдрпоу", "єдр";
+triggers startend => "ЄДРПОУ","ЄДР","edrpou", "єдрпоу", "єдр";
```
Comment (Vanuan) [inline lib/DDG/Spice/Edrpou.pm]: These first 2 are also upper case. As long as it supports uppercasing cyrillic? Maybe it doesn't

File: lib/DDG/Spice/Edrpou.pm [hunk 2]
```diff
@@ -15,12 +15,12 @@ spice wrap_jsonp_callback => 1;
...
 
 # Handle statement
 handle remainder => sub {
-    # Trigger if query includes an 8 digit number
-    return $_ if $_ =~ /\d{8}/;
+    # Trigger if query includes number length 5-8
+    return $_ if $_ =~ /^\d{5,8}$/;
```
Comment (Vanuan) [inline lib/DDG/Spice/Edrpou.pm]: Is it correct? Isn't it 5-8? Or maybe `{5,6,7,8}`?

Comment (kirkins) [inline lib/DDG/Spice/Edrpou.pm]: It works I tested.

File: lib/DDG/Spice/Edrpou.pm (hunk 1 above)
Comment (kirkins) [inline lib/DDG/Spice/Edrpou.pm]: I'm not sure 🤔

File: lib/DDG/Spice/Edrpou.pm (hunk 2 above)
Comment (Vanuan) [inline lib/DDG/Spice/Edrpou.pm]: cool

**Pull request merged**